common/src/main/scala/common/validation/Validation.scala

```diff
@@ -47,6 +47,10 @@ object Validation {
       Validated.fromTry(t).leftMap(_.getMessage).toValidatedNel[String, A] 
     }
 
+    def toErrorOrWithContext(context: String): ErrorOr[A] = {
+      Validated.fromTry(t).leftMap(e => context + ": " + e.getMessage).toValidatedNel[String, A]
```

Comment:
is it against the ErrorOr rules to grab the stack trace too? 😛

Reply:
this looks like `contextualizeErrors` to me